app/connectors/des/DesConnector.scala

```diff
@@ -60,13 +58,12 @@ class DesConnector @Inject() (servicesConfig: ServicesConfig, httpClient: HttpCl
     httpClient.GET[Option[FinancialData]](getFinancialURL)
   }
 
-  //this may only be needed for welsh indicator if needed
-  def getCustomerData(vrn: String): Future[CustomerInformation] = {
+  def getCustomerData(vrn: String): Future[Option[CustomerInformation]] = {
```

Comment:
Let's create Vrn type and use it instead of Strings. See `pay-api` :

```
import play.api.libs.functional.syntax._
import play.api.libs.json.Format

/**
 * Vat Registration Number (Vrn)
 */
final case class Vrn(value: String)

object Vrn {

  implicit val format: Format[Vrn] = implicitly[Format[String]].inmap(Vrn(_), _.value)

}

```

Reply:
done